docs/platform/what-is-tizen/versions/tizen-6-0-m2.md

```diff
@@ -0,0 +1,96 @@
+- **TizenFX**:
+    - Tizen platform-specific API set
+    - API level 8 released
+- **Xamarin**:
+    -  Latest Xamarin.Forms (4.8.0)  and Xamarin.Essentials (1.5.3) are available.
+    - TV UI Controls and CircularUI: New UI controls for Smart TVs and Smart Watches.
+    - XAML Hot Reload: Boosting developer productivity
+
+### Web framework
+
+Tizen 6.0 provides a new user experience through device-to-device connectivity and interworking.
+
+- Web-based device offloading services
+
+As the complexity of applications in the web environment increases, a method of maximizing computational power is needed.
+
+Tizen 6.0 provides **multi-device offloading service** technology using multiple devices:
+
+![img](media/6.0_1_web.png)
+
+### Lightweight Web Solution
+
+Escargot is the Tizen JavaScript engine for the product having hardware resource restriction. It uses about 14% in binary size and 35% in memory usage for the benchmark application compared to those of Chrome's V8 JavaScript engine. Escargot supports ES6 full specification and also supports ES 10 up to a large extent:
+
+ The following are the details for memory size and features in comparison to Chrome V8 benchmarks:
+
+  - Memory size  (vs chrome v8) octane benchmark in Ubuntu 18.04
+    - Binary size : 13.7 %
+    - Average RSS : 35.1 %
+
+  - Features
+    - Support ES6 Full feature and partial support of ES10
+
+## Machine Learning framework
+
+NNSteamer is an on-device AI framework that makes it easy to configure complex artificial neural networks in the application by making artificial neural network configurations with Gstreamer's plugin. 	 
+
+In Tizen 6.0, NN Runtime supports control-flow and dynamic tensor that enables voice model acceleration. In addition, by providing the NN Compiler front-end for NN Runtime, it can convert well-known models such as Tensorflow and TensorFlow-lite to be used on CE devices with insufficient resources on NN Runtime.	 
+
+- NNStreamer (Neural Network Pipeline)
+  - .NET and C APIs supported. 	 
+  - Various accelerating hardware and frameworks supported. 	 	 
+  - Implement AI app more easily. 	 
+  - Execute AI app more efficiently. 	 
+
+- NNTrainer (On-Device training)
+  - Transfer learning, meta learning, and reinforcement learning. 	 
+  - APIs to construct and update neural network layers.
+
+- NN Runtime 	 
+  - Tizen’s default Neural Network inference framework (since v5.0).
+  - CPU and GPU mixed acceleration with profiling based optimal scheduling (since v5.5).
+  - Useful features such as Control-flow, Dynamic Tensor, Shape Inference, and so forth for speech model support.
+
+- NN Compiler in Tizen SDK 	 
+  - Support well-known ML frameworks and model formats such as TensorFlow, TensorFlow-lite (preferential), Caffe, and ONNX (experimental).
+
+   **Figure: NN Runtime**
+
+   ![NN Runtime](./media/6.0_1_NNRuntime.png)
+
+   **Figure: NN Streamer**
+
+   ![NN Streamer](./media/6.0_1_NNStreamer.png)
+
+### Core and Kernel
```

Comment:
![image](https://user-images.githubusercontent.com/34763817/122014890-3d9b8180-cddd-11eb-95a8-e12b60ca08d5.png)

Should the heading Core and Kernel be at same level as Machine Learning?

Also the first heading # Enhancement ... seems too long. This is an optional comment can it be made shorter?

Reply:
@jhchoilee Please resolve the comment. Thanks.

Reply:
How about remove this line "Enhancement of development environment and core component"
and
change 
from ### Tizen.NET to ## Tizen.NET,
from ### Web framework to ## Web framework,
from ### Lightweight Web Solution to ## Lightweight Web Solution
from ### Core and Kernel to ## Core and Kernel

Please let me know your comment.
Based on your reply, I will commit.

Reply:
@jhchoilee you are suggesting the hierarchical change in headings. 
IMO, if  "Enhancement of development environment and core component" can be omitted without altering the message that you wish to convey. Then it is fine. 

We can write : Developement environment and core components enhancement. , But still it is long. So again if ommiting this won't hamper the message, then we can.

Reply:
I emoved this line "Enhancement of development environment and core component"
and
changed
from ### Tizen.NET to ## Tizen.NET,
from ### Web framework to ## Web framework,
from ### Lightweight Web Solution to ## Lightweight Web Solution
from ### Core and Kernel to ## Core and Kernel
with new commit